org.civicrm.reqangular/src/common/directives/timepicker-select.directive.js

```diff
@@ -0,0 +1,79 @@
+/* eslint-env amd */
+
+define([
+  'common/modules/directives'
+], function (directives) {
+  'use strict';
+
+  directives.directive('timepickerSelect', ['$templateCache', function ($templateCache) {
+    return {
+      scope: {
+        timepickerSelectDefaultOption: '<',
```

Comment:
rename to placeholder and make it a `@` binding instead:

`<select timepicker-select timepicker-select-placeholder="Please select a time..."></select>`